### DISCUSSION TITLE
Clouds on maps

### DISCUSSION THREAD

**Discussion by feh123** [SerBrynden/PiClock#14] [General]
Hi

I have been using PiClock and it's working very well. A great piece of software! My weather provider is Tomorrow and I use The Weather Channel colour code. I was wondering how to show the cloud level too. Currently clouds would not show up on my setup. Any thoughts about viewing the cloud base on the maps much appreciated!

**Comment by SerBrynden** [comment]
That's a limitation of the RainViewer API.  It can only show radar or satellite infrared (clouds), but not both at the same time.  The DarkSky API that we previously used could do that, but RainViewer can't.

**Comment by feh123** [comment]
Ah - thanks. Rainviewer do a great job so I cannot complain! I do have a iOS weather app called weather&radar that does provide this so I do know what the cloud cover will be.
